docs/product-manuals/clients/cli-client/index.md

```diff
@@ -38,13 +38,13 @@ zbctl [options] [command]
 ```
 
 ```
-zbctl is command line interface designed to create and read resources inside zeebe broker.
-It is designed for regular maintenance jobs such as:
-        * deploying processes,
-        * creating jobs and process instances
-        * activating, completing or failing jobs
-        * update variables and retries
-        * view cluster status
+zbctl is command line interface designed to create and read resources inside Zeebe broker.
```

Comment:
```suggestion
`zbctl` is command line interface designed to create and read resources inside the Zeebe broker.
```

Please add zbctl to the glossary and style guide, similar to a product name that we capitalize zbctl is an except that should be written like code.

Reply:
I just realized this is actually output from the CLI itself - https://docs.camunda.io/docs/product-manuals/clients/cli-client/index

Can you work with @jwulf to update the [output of the CLI](https://github.com/camunda-community-hub/zbctl-via-npm)?

Reply:
Adding this to the glossary and style guide. Also, just reached out to @jwulf on this. I don't think we're able to add a code block inside of a code block here. What would be the best practice?

Reply:
Yeah, the triple backtick makes the `zbctl` monospaced. 

The only thing to change in the CLI output would be to add an article to it, either _a_ or _the_: 

```
zbctl is a command line interface...
```

That change needs to be done in the Zeebe repo itself.

Reply:
Ref: https://github.com/camunda-cloud/zeebe/pull/7558

Reply:
Done and done ✅